client/account-recovery/reset-password-confirm-form/index.jsx

```diff
@@ -60,6 +68,8 @@ class ResetPasswordConfirmForm extends Component {
 
 		const { newPassword } = this.state;
 
+		const capturePasswordInputRef = ( passwordInput ) => this.passwordInput = passwordInput;
```

Comment:
Since this isn't the actual password shouldn't this be `const capturePasswordInputRef = ( passwordInputRef ) => this.passwordInputRef = passwordInputRef;`

Reply:
That's a good idea. Updated in 79e6bb0.